File: Java/Day 4 Packages/Assignment 4_1/office/Date.java
package office.utility;

public class Date{


	private int dd,yy,mm;
	
	public Date(){
		dd=mm=1;
		yy=2023;
	}
	
	public Date(int dd,int mm,int yy){
		this.dd = dd;
		this.mm = mm;
		this.yy = yy;
	}

	public void display(){
		
		System.out.println(dd+"/"+yy+"/"+mm);
	
	}








}

File: cpp/CPP Mam/ReusingClasses/Person/Date.cpp
#include"date.h"

Date::Date()
{
    day=month=1;
    year=2023;
}
 Date::Date(int d,int m,int y)
 {
    day=d;
    month=m;
    year=y;
 }
void Date::Display()const
{
    cout<<"\n\t Date    : "<<day<<"/"<<month<<"/"<<year;
}